src/options.html

```diff
@@ -13,15 +13,8 @@ <h1>Xdebug Extension</h1>
     <form>
         <fieldset>
             <legend>Options</legend>
-            <label for="idekey">IDE Key</label>
-            <input type="text" id="idekey" placeholder="IDE key" list="idekeys" value="">
-            <datalist id="idekeys">
-                <option value="PHPSTORM">PhpStorm</option>
-                <option value="XDEBUG_ECLIPSE">Eclipse</option>
-                <option value="netbeans-xdebug">NetBeans</option>
-                <option value="macgdbp">MacGDBp</option>
-                <option value="VSCODE">VSCode</option>
-            </datalist>
+            <label for="debugtrigger">Debug Trigger or Xdebug Cloud Key</label>
+            <input type="text" id="debugtrigger" placeholder="Debug trigger or Xdebug Cloud Key" value="">

```

Comment:
Should the placeholder perhaps also be `YOUR-NAME` - to show the default?

Reply:
That's a good plan. I have pushed that change.